docs/csharp/tour-of-csharp/classes-and-objects.md

```diff
@@ -63,9 +63,9 @@ Each member of a class has an associated accessibility, which controls the regio
 * `protected`
 	- Access limited to this class or classes derived from this class
 * `internal`
-	- Access limited to this program
+	- Access limited to this Assembly (.exe, .dll etc.)
```

Comment:
better: "the current assembly (.exe, .dll, etc.)" Note that assembly should not be capitalized.